ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_linalg.py

```diff
@@ -38,6 +38,30 @@ def test_jax_numpy_det(dtype_and_x, as_variable, native_array, num_positional_ar
     )
 
 
+# eig
+@handle_cmd_line_args
+@given(
+    dtype_and_x=_get_dtype_and_matrix(),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.jax.numpy.linalg.eig"
+    ),
+)
+def test_jax_numpy_eig(dtype_and_x, as_variable, native_array, num_positional_args):
+    dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=dtype,
+        as_variable_flags=as_variable,
+        with_out=False,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        frontend="jax",
+        fn_tree="numpy.linalg.eig",
+        rtol=1e-04,
+        atol=1e-04,
+        a=np.asarray(x[0], dtype=dtype[0]),
```

Comment:
we actually don't use `np.asarray` anymore for inputs, you can just remove this

Reply:
sure thank you